stats/includes/team.inc

```diff
@@ -305,7 +302,7 @@ function showTeamMbrs($curTeam, $tally_name)
     $req_team_id = get_integer_param($_GET, 'tid', null, 0, null);
 
     $subtitle = null;
-    $latestMbr = mysqli_query(DPDatabase::get_connection(), "SELECT users.username AS Username, users.u_privacy AS Privacy FROM user_teams INNER JOIN users ON user_teams.latestUser = users.u_id WHERE user_teams.id = ".$curTeam['id']." AND user_teams.latestUser <> 0");
+    $latestMbr = DPDatabase::query("SELECT users.username AS Username, users.u_privacy AS Privacy FROM user_teams INNER JOIN users ON user_teams.latestUser = users.u_id WHERE user_teams.id = ".$curTeam['id']." AND user_teams.latestUser <> 0");
```

Comment:
curTeam['id'] is an integer and doesn't need encoding, right? Should we use sprintf %d here to be safer?

Reply:
It is. It should have `sprintf()` done on it but I opted to not do that lift on the team or member code which is abysmal. 😦